core/dive.h

```diff
@@ -76,7 +76,7 @@ extern "C" {
 
 extern int last_xml_version;
 
-enum dive_comp_type {OC, CCR, PSCR, FREEDIVE, NUM_DC_TYPE};	// Flags (Open-circuit and Closed-circuit-rebreather) for setting dive computer type
+enum divemode_t {OC, CCR, PSCR, FREEDIVE, NUM_DC_TYPE, UNDEF_COMP_TYPE};	// Flags (Open-circuit and Closed-circuit-rebreather) for setting dive computer type
```

Comment:
So NUM_DC_TYPE is intended to be last (as it simply gives you the number of types that we have and can be used as loop delimiter... it would make sense to rename this to NUM_DIVEMODE_TYPE as well